### ISSUE TITLE
Dependencies and documentation

### ISSUE THREAD

**Issue by mjafin** [data61/gossamer#5]
Hi there,
Many thanks for releasing gossamer to the community. I had a few issues trying to use it, namely the documentation is quite sparse and in a typical HPC environment the dependencies are quite restricting. We use CentOS6 and e.g. libboost is `1.41.0` so gossamer won't compile. 

Any chance you could release precompiled binaries compiled on older servers? Like what bioconda are doing, http://bioconda.github.io/

**Comment by axman6** [comment]
@Deguerre Any thoughts on this?

**Comment by Deguerre** [comment]
First step is to implement a statically-linked target. Happy to do that, but it probably won't get done before Christmas.

**Comment by mjafin** [comment]
Thanks for the comments - will the statically linked target still depend on quite a new libc though if you compile it on Ubuntu 16?

**Comment by Deguerre** [comment]
No, that won't be an issue. The problem is the C++ ABI, which only affects C++ libraries, which libc is not.

**Comment by Sinan81** [comment]
A coworker of mine successfully compiled gossamer on Centos6. I believe the key to the successful compilations was using boost 1.55 compiled with extra required libraries as discussed in http://www.boost.org/doc/libs/1_55_0/more/getting_started/unix-variants.html
